packages/debian/metalk8s-sosreport/debian/src/containerd.py

```diff
@@ -0,0 +1,73 @@
+#! /bin/env python3
```

Comment:
Don't think we still need these, since they also are under a `common` directory, right?

Reply:
These files (containerd.py and metalk8s.py) should have been deleted after tests for the common folder.
I will take care of it.

Reply:
(not related to the shebang sorry)